classes/ActionScheduler_Retry.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * Class ActionScheduler_Retry
+ */
+class ActionScheduler_Retry {
```

Comment:
should this class be integrated into `ActionScheduler_Action`. `retry` looks like a property of an action instead of a separate entity. wdyt?